Assets/FishFactory/Scripts/FactoryFishSpawner.cs

```diff
@@ -208,6 +238,23 @@ private FactoryFishState.State RandomizeFishState()
 
         FactoryFishState.State state;
 
+        if (toggleFishGuttingChance)
+        {
+            if (randomValue <= successfullGuttingChance)
+            {
+                state = FactoryFishState.State.GuttingSuccess;
+            }
+            else if (randomValue <= successfullGuttingChance + incompleteGuttingChance)
+            {
+                state = FactoryFishState.State.GuttingIncomplete;
```

Comment:
Isn't gutting incomplete and failure the same thing? Both are bad guttings of fish?

Reply:
there are 3 statesincomplete are sent to a different conveyor while failure are so bad they are thrown in the bin

Reply:
Ok sounds good.